code/game/machinery/Sleeper.dm

```diff
@@ -26,6 +26,9 @@
 	var/disallow_occupant_types = list()
 	var/display_loading_message = TRUE
 
+	// rate at which alcohol is removed per tick (see process())
```

Comment:
```suggestion
	/// rate at which alcohol is removed per second (see process())
```
// is a standard comment
/// is a special kind of comment read by VSCode and IDEs with intellisense, that makes it so that mousing over the variable will include a popup with your comment. We use the /// version for variables set on the object to help keep the code more readable. 

It's also possible to make multi-line popups to explain more complicated vars to future contributors.
/**
 * This is a multi-line documentation
 * And this is its second line
 * And its third line
 */